src/crate/theme/rtd/crate/navbar.html

```diff
@@ -15,32 +15,40 @@
             <a href="https://crate.io/products/">Products</a>
             <ul class="sub-menu">
               <li class="menu-item"><a href="https://crate.io/products/cratedb/">CrateDB</a></li>
-              <li class="menu-item"><a href="https://crate.io/products/cratedb-cloud-on-azure/">CrateDB Cloud on Azure</a></li>
               <li class="menu-item"><a href="https://crate.io/products/cratedb-cloud/">CrateDB Cloud</a></li>
-              <li class="menu-item"><a href="https://crate.io/products/crate-iot-data-platform/">Crate IoT Data Platform</a></li>
+              <li class="menu-item"><a href="https://crate.io/docs/">Get started</a></li>
             </ul>
           </li>
           <li class="navlink w-nav-link menu-item menu-item-has-children">
-            <a href="https://crate.io/use-cases/">Use Cases</a>
+            <a href="https://crate.io/customers/">Customers</a>
             <ul class="sub-menu">
-              <li class="menu-item"><a href="https://crate.io/customers/">Customers</a></li>
-              <li class="menu-item"><a href="https://crate.io/use-cases/iot-sensor-data/">IoT &amp; Sensor Data</a></li>
-              <li class="menu-item"><a href="https://crate.io/use-cases/time-series/">Time Series</a></li>
-              <li class="menu-item"><a href="https://crate.io/use-cases/geospatial-analytics/">Geospatial Analytics</a></li>
-              <li class="menu-item"><a href="https://crate.io/use-cases/cyber-security/">Cybersecurity</a></li>
+              <li class="menu-item"><a href="https://crate.io/use-cases/">Use Cases</a></li>
             </ul>
           </li>
+          <li class="navlink w-nav-link menu-item"><a href="https://crate.io/partner/">Partners</a></li>
+           <li class="navlink w-nav-link menu-item menu-item-has-children">
+            <a href="https://crate.io/about/">Company</a>
+            <ul class="sub-menu">
+              <li class="menu-item"><a href="https://crate.io/jobs/">Careers</a></li>
+              <li class="menu-item"><a href="https://crate.io/a/press-category/in-the-news/">Newsroom</a></li>
+              <li class="menu-item"><a href="https://crate.io/events/">Events</a></li>
+              <li class="menu-item"><a href="https://crate.io/a/tag/awards/">Awards</a></li>
+              <li class="menu-item"><a href="https://crate.io/contact/"></a>Contact us</li>
+             </ul>
+          </li>
           <li class="navlink w-nav-link menu-item menu-item-has-children">
             <a href="https://crate.io/resources/">Resources</a>
             <ul class="sub-menu">
-              <li class="menu-item"><a href="https://crate.io/docs/">Documentation</a></li>
-              <li class="menu-item"><a href="https://crate.io/resources/">Resource Library</a></li>
+              <li class="menu-item"><a href="">Webinars</a></li>
```

Comment:
this line shouldn't be here i think @scjimenez

Reply:
You are so right, thanks!